packages/meteor-wallet/tsconfig.lib.json

```diff
@@ -6,5 +6,16 @@
     "types": []
   },
   "include": ["**/*.ts"],
-  "exclude": ["**/*.spec.ts"]
+  "exclude": [
+    "jest.config.ts",
+    "**/*.test.ts",
+    "**/*.spec.ts",
+    "**/*.test.tsx",
+    "**/*.spec.tsx",
+    "**/*.test.js",
+    "**/*.spec.js",
+    "**/*.test.jsx",
+    "**/*.spec.jsx",
+    "**/*.d.ts"
+  ]
```

Comment:
As with the other packages, we should be able to condense this to `"exclude": ["jest.config.ts", "**/*.spec.ts"]`